pkg/proxy/proxyports/proxyports_test.go

```diff
@@ -112,7 +112,7 @@ func TestPortAllocator(t *testing.T) {
 	// 2nd release decreases the count to zero
 	err = p.releaseProxyPort("listener1", time.Microsecond)
 	require.NoError(t, err)
-	time.Sleep(time.Millisecond)
+	time.Sleep(5 * time.Millisecond)
```

Comment:
How about something like this:
```
	require.Eventually(t, func() bool {
		return pp.nRedirects == 0
	}, 100*time.Millisecond, time.Millisecond)
```
:thinking: ?

Reply:
Even better!